tests/test_data_management.py

```diff
@@ -92,20 +92,30 @@ def test_build_method_with_new_drivers(self):
 
     def test_platform_serialization_from_imported_dict(self):
         """Test platform serialization by building a platform, saving it and then load it back again twice. Starting from a given dict."""
-        original_platform = ql.build_platform(Galadriel.runcard)
+        original_dict = Galadriel.runcard
+        # Check that the new serialization with ruamel.yaml.YAML().dump works for different formats...
+        original_dict["gates_settings"]["gates"]["Y(0)"][0]["pulse"]["phase"] = 1.6707963267948966  # Test long decimals
+        original_dict["instruments"][0]["awg_sequencers"][0]["intermediate_frequency"] = 100_000_000  # Test underscores
+        original_dict["instruments"][1]["awg_sequencers"][0]["sampling_rate"] = 7.24730e09  # Test scientific notation
+        original_dict["instruments"][4]["firmware"] = None  # Test None values
+
+        original_platform = ql.build_platform(original_dict)
+
         path = save_platform(path="./test.yml", platform=original_platform)
         saved_platform = ql.build_platform(path)
+
         new_path = save_platform(path="./test.yml", platform=saved_platform)
         new_saved_platform = ql.build_platform(new_path)
 
-        with open(file="examples/runcards/galadriel.yml", mode="r", encoding="utf8") as yaml_f:
```

Comment:
This file doesn't exist, how is this even working?

Reply:
Same thing for the test below

Reply:
What do you mean it doesn't exist? It is there: 
https://github.com/qilimanjaro-tech/qililab/blob/main/examples/runcards/galadriel.yml

Reply:
The thing is, in this test we were testing `Galadriel.runcard` from `data.py`, and we just also read the `yaml` in `galadriel.yaml`, to then check that they are not the same... but that doesn't make any sense to test..

Reply:
Ok my bad.

Reply:
I don't like at all using files outside the `test` folder inside tests. Good thing you deleted it

Reply:
> Like why would they be different? I mean they are, because they are to different dictionaries, but it doesn't;t make sense to add here a line for the sake of checking that, since that is no functionality at all 🙃

Completely agree!